docs/gatsby-config.js

```diff
@@ -16,29 +16,43 @@ module.exports = {
           1: 'version-1',
         },
         sidebarCategories: {
-          null: ['index', 'getting-started', 'whats-new'],
-          Essentials: [
-            'essentials/schema',
-            'essentials/server',
-            'essentials/data',
+          null: [
+            'index',
+            'getting-started',
+            '[Changelog](https://github.com/apollographql/apollo-server/blob/master/CHANGELOG.md)',
           ],
-          Features: [
-            'features/caching',
-            'features/mocking',
-            'features/errors',
-            'features/data-sources',
-            'features/subscriptions',
-            'features/metrics',
-            'features/graphql-playground',
-            'features/scalars-enums',
-            'features/unions-interfaces',
-            'features/directives',
-            'features/creating-directives',
-            'features/authentication',
-            'features/testing',
-            'features/apq',
-            'features/health-checks',
-            'features/file-uploads',
+          'Defining a Schema': [
+            'defining-a-schema/schema',
+            'defining-a-schema/scalars-enums',
+            'defining-a-schema/unions-interfaces',
+            'defining-a-schema/directives',
+            'defining-a-schema/creating-directives',
+          ],
+          'Resolving Requests': [
+            'resolving-requests/data',
+            'resolving-requests/data-sources',
+            'resolving-requests/errors',
+            'resolving-requests/authentication',
+            'resolving-requests/file-uploads',
+            'resolving-requests/subscriptions',
+          ],
+          'Testing & Monitoring': [
+            'testing-and-monitoring/mocking',
+            'testing-and-monitoring/graphql-playground',
```

Comment:
I'm not sure that GraphQL Playground fits into either _testing_ or _monitoring_, but I can see how it is currently here.  To me, it seems like an important earlier-stage consideration in this IA and I want to suggest that this goes into the "Resolving Requests" category (keeping in mind my other comment about another name for that.)